Dockerfile

```diff
@@ -1,11 +1,10 @@
 FROM golang:1.8.3-alpine3.6 AS builder
 COPY ./ /go/src/github.com/pires/nats-operator
 WORKDIR /go/src/github.com/pires/nats-operator
-RUN \
-	apk add --update git; \
-	go get -u github.com/golang/dep/cmd/dep; \
-	dep ensure; \
-	go build -ldflags '-w -extldflags=-static' -o /nats-operator ./cmd/operator/main.go
+RUN apk add --update git
+RUN go get -u github.com/golang/dep/cmd/dep
+RUN dep ensure
+RUN CGO_ENABLED=0 go build -installsuffix cgo -o /nats-operator ./cmd/operator/main.go
```

Comment:
Mainly because this is an intermediate container — so we don't have to worry about layers and image size — and I believe it improves readability.